packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -138,10 +137,12 @@ async function runPipelineTransformStep(
         // @ts-ignore: Deprecated
         urlPath: `./${path.basename(rootFileName + destExt)}`,
       });
+
+      // if step returned a value, only update code (don’t touch .map)
       if (typeof result === 'string') {
-        output[destExt] = {code: result, map};
+        output[destExt].code = result;
       } else if (result && typeof result === 'object' && (result as {result: string}).result) {
-        output[destExt] = {code: (result as {result: string}).result, map};
+        output[destExt].code = (result as {result: string}).result;
```

Comment:
After just merging the source mapping PR, I wanted to do a little cleanup here. This doesn’t change execution, but it’s a little safer. I’ve found that source maps were easy to drop in the transformation step here.